fairworkflows/fairstep.py

```diff
@@ -77,16 +77,52 @@ def is_pplan_step(self):
         """Return True if this FairStep is a pplan:Step, else False."""
         return (self.self_ref, RDF.type, namespaces.PPLAN.Step) in self._rdf
 
+    @is_pplan_step.setter
+    def is_pplan_step(self, value:bool):
+        """
+        Adds/removes the pplan:Step triple from the RDF, in accordance with the provided boolean.
+        """
+        if value is True:
+            if self.is_pplan_step is False:
+                self.set_attribute(RDF.type, namespaces.PPLAN.Step, overwrite=False) 
+        elif value is False:
+            if self.is_pplan_step is True:
+                self.remove_attribute(RDF.type, object=namespaces.PPLAN.Step)
+
     @property
     def is_manual_task(self):
         """Returns True if this FairStep is a bpmn:ManualTask, else False."""
         return (self.self_ref, RDF.type, namespaces.BPMN.ManualTask) in self._rdf
 
+    @is_manual_task.setter
+    def is_manual_task(self, value:bool):
```

Comment:
Are manual_task and script_task actually mutually exclusive? If so we should raise an issue for validating that and adapting RDF accordingly when setting one of the 2.

Reply:
They are actually (see validation). They are now automatically mutually exclusive.